ang/civicase/FileUploader.js

```diff
@@ -46,50 +57,46 @@
       }
     });
 
-    $scope.deleteActivity = function deleteActivity() {
+    $scope.deleteActivity = function deleteActivity () {
       $scope.uploader.clearQueue();
       $scope.activity = null;
     };
 
-    $scope.saveActivity = function saveActivity() {
+    $scope.saveActivity = function saveActivity () {
       var promise = crmApi('Activity', 'create', $scope.activity)
         .then(function (r) {
           var target = {entity_table: 'civicrm_activity', entity_id: r.id};
           _.each($scope.uploader.getNotUploadedItems(), function (item) {
             item.formData = [_.extend({crm_attachment_token: CRM.crmAttachment.token}, target, item.crmData)];
           });
           return $scope.uploader.uploadAllWithPromise();
-        }).then(function(){
+        }).then(function () {
           return pwait(1000); // Let the user absorb what happened.
-        }).then(function(){
+        }).then(function () {
           $scope.uploader.clearQueue();
           initActivity();
           if ($scope.onUpload) {
             $scope.$parent.$eval($scope.onUpload);
           }
         });
-      return block(crmStatus({start: ts('Uploading...'), success: ts('Uploaded')}, promise));
+
+      return $scope.block(crmStatus({start: $scope.ts('Uploading...'), success: $scope.ts('Uploaded')}, promise));
     };
 
+    function initActivity () {
+      $scope.activity = {
+        case_id: $scope.ctx.id,
+        activity_type_id: 'File Upload',
+        subject: ''
+      };
+    }
+
     // TODO: Test interrupted transfer.
 
-    function pwait(delay) {
+    function pwait (delay) {
```

Comment:
Whats `pwait`? please have a more meaningful name. 
Do we need the TODO? if yes, there should not be any line break.

Reply:
`pwait` was something that was coming from the previous code. So not sure what this do and therefore can't change the name.
Removed the space however.